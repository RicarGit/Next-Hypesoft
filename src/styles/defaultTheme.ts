import { DefaultTheme } from "styled-components"

const defaultTheme: DefaultTheme = {
  colors: {
    primary: '#242424',
    secondary: '#ED1CA6',
    sec: '#7D00FF',
    white: '#FFFFFF',
    gray: {
      100: '#DFE3EB',
      500: '#A5A5A5',
    },
  },
  font: {
    family: 'Inter',
    weigths: {
      regular: 400,
      medium: 500,
      semiBold: 600,
      bold: 700
    },
  },
}

export default defaultTheme