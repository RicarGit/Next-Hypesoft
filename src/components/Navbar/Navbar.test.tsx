import { render } from '@testing-library/react'
import { ThemeProvider } from 'styled-components'
import defaultTheme from 'styles/defaultTheme'
import NavBar from '.'

describe('NavBar', () => {
  it('should match snapshot', () => {
    const { container } = render(
      <ThemeProvider theme={defaultTheme}>
        <NavBar />
      </ThemeProvider>
    )

    expect(container.firstChild).toMatchSnapshot()
  })
})