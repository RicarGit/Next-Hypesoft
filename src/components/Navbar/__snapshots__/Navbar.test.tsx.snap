// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar should match snapshot 1`] = `
<nav
  class="sc-bczRLJ fCxCre"
>
  <a
    class="sc-gsnTZi kBWNga"
    href="/#about"
  >
    about
  </a>
  <a
    class="sc-gsnTZi kBWNga"
    href="/#services"
  >
    services
  </a>
  <a
    class="sc-gsnTZi kBWNga"
    href="/#works"
  >
    works
  </a>
  <a
    class="sc-gsnTZi kBWNga"
    href="/#contact"
  >
    contact
  </a>
</nav>
`;
